src/app/home/room-creation-modal/room-creation-modal.component.scss

```diff
@@ -0,0 +1,23 @@
+.form-container .mat-form-field {
+  width: 100%;
+}
+
+.form-container form {
+  margin-bottom: 2rem;
+}
+
+.form-container form > * {
+  margin: .5rem 0;
+}
+
+.form-container .mat-radio-button {
+  margin: 0 1.2rem;
+}
+
+.full-width {
+  width: 100%;
+}
+
+.half-width {
+  width: 48%;
+}
```

Comment:
Very dishonest class name :) I'm fine with it though, just made me chuckle

Reply:
True, i change it